French/e107_handlers/tiny_mce/plugins/ibrowser/langs/French.php

```diff
@@ -0,0 +1,57 @@
+<?php
```

Comment:
TinyMce has not been located in e107_handlers for a very long time. ibrowser is no longer in use.